cinema/serializers.py

```diff
@@ -18,7 +18,69 @@ def update(self, instance, validated_data):
             "description", instance.description
         )
         instance.duration = validated_data.get("duration", instance.duration)
+        instance.save()
+
+        return instance
+
+
+class ActorSerializer(serializers.Serializer):
```

Comment:
It could be shorter if you use ModelSerializer.

Reply:
Yes, thanks, but when i passed this task i didn't know about ModelSerializer, we just study it in the next lesson.